### DISCUSSION TITLE
A marker/icon where the memory slot is in use?

### DISCUSSION THREAD

**Discussion by Overland-DX** [PE5PVB/TEF6686_ESP32#441] [Ideas]
I have several times saved over another channel that is in use. Is it possible to show in any way that the memory slot is already in use?
Like for example if I try to save something on #16 and have forgotten that I have already saved some stations on #16 and #17, then an icon or something appears.

And yes! I have an extremely bad memory! :D

**Comment by PE5PVB** [comment]
A memory "skip" function is planned for a future release.

**Comment by Overland-DX** [comment]
It's fantastic :) Then we'll have to wait and see what time brings :)

**Comment by Overland-DX** [comment]
Thank you so much @ohmytime  🥇

**Comment by Overland-DX** [comment]
This feature was exactly what I wanted. Thanks to all involved.

**Comment by ohmytime** [comment]
> This feature was exactly what I wanted. Thanks to all involved.

You're welcome
